caom2/caom2/observation.py

```diff
@@ -640,7 +636,7 @@ def __init__(self,
               planes=planes,
               environment=environment,
               target_position=target_position)
-        self._members = caom_util.TypedSet(ObservationURI, )
+        self._members = caom_util.TypedSet(str, )
```

Comment:
Is the `TypedSet` still useful for an `str`?

Reply:
To make sure people don't pass by mistake the observation object or now `ObservationURI`.